packages/php-wasm/universal/src/lib/universal-php.ts

```diff
@@ -256,6 +256,15 @@ export interface IsomorphicLocalPHP extends RequestHandler {
 	 */
 	mv(oldPath: string, newPath: string): void;
 
+	/**
+	 * Copies a file or directory in the PHP filesystem to a
+	 * new location.
+	 *
+	 * @param oldPath The path to rename.
+	 * @param newPath The new path.
```

Comment:
Is this a path to the parent directory? Or does it have to include a file name?

Reply:
@adamziel You need to provide the full, new path. I made it more explicit:

https://github.com/WordPress/wordpress-playground/pull/846/files#diff-eeec0049f1f178834acbd06e83e370eb8044a650901eb4dd3f9189126a07460eR264